Request/ParamFetcher.php

```diff
@@ -136,7 +136,7 @@ protected function cleanParamWithRequirements(ParamInterface $param, $paramValue
         }
         if (null === $this->validator) {
             throw new \RuntimeException(
-                'The ParamFetcher requirements feature requires the symfony/validator component.'
+                'The ParamFetcher requirements feature requires the validator service.'
```

Comment:
I think we can improve this a bit by triggering a deprecation in the constructor when no validator is passed. That would allow us to get rid of this runtime check in the next major version.

Reply:
But if you prefer to, we can do that in a follow-up PR.

Reply:
Done in #1980 but you've already seen it :)